.github/workflows/main.yml

```diff
@@ -50,10 +50,58 @@ jobs:
           name: AppImage
           path: linuxdeploy*.AppImage*
 
+  container_build:
+    # The host should always be linux
+    runs-on: ubuntu-latest
+    name: ${{ matrix.BUILD_TYPE }} ${{ matrix.arch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        BUILD_TYPE: [appimage, coverage]
+        distro:
+         - bullseye
+        arch:
+          - aarch64
+          - armv7
+          - armv6
+          - ppc64le
+          - s390x
+    steps:
+      - uses: actions/checkout@v2.1.0
+        with:
+          submodules: recursive
+      - uses: uraimo/run-on-arch-action@v2.1.1
```

Comment:
As explained on IRC, third party actions are a no-go in the supply chain. Have had very bad experiences with this kind of stuff. It limits the portability of the build scripts (i.e., I can't move to another CI service by just writing a YAML file), and it doesn't allow for testing the CI toolchain locally (unlike a script with a Dockerfile next to it).